docs/src/examples/components/Popup/Usage/PopupExampleCloseButton.shorthand.steps.ts

```diff
@@ -0,0 +1,23 @@
+import { Dropdown, Popup } from '@stardust-ui/react'
+
+const selectors = {
+  toggleIndicator: `.${Dropdown.slotClassNames.toggleIndicator}`,
+  item: (itemIndex: number) => `.${Dropdown.slotClassNames.itemsList} li:nth-child(${itemIndex})`,
+  popupTrigger: `.${Popup.slotClassNames.trigger}`,
+}
+
+const config: ScreenerTestsConfig = {
+  themes: ['teams', 'teamsDark', 'teamsHighContrast'],
+  steps: [
+    (builder, keys) =>
+      builder
+        .click(selectors.popupTrigger)
```

Comment:
https://github.com/stardust-ui/react/pull/1669/files#r305348228

You can use `Button.className` instead